pkg/kn/commands/domain/domain_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright © 2021 The Knative Authors
```

Comment:
This file has no `Test..()` function(s)? So essentially a no-op

Reply:
Done in https://github.com/knative/client/pull/1267/commits/928e2cfb657dbb8011019125942a2270fc8614e7.